### ISSUE TITLE
Better default height for metrics property

### ISSUE THREAD

**Issue by FlorianJacta** [Avaiga/taipy#2142]
### Description

The height of a metric is taking more than half the available height of my page. This will never be a correct value. (I would also argue the same for the width) A new default height should be defined.

```python
from taipy.gui import Gui
import taipy.gui.builder as tgb

value = 50

with tgb.Page() as page:
    tgb.metric("{value}", type="none")

Gui(page=page).run(title="Frontend Demo")

```

![image](https://github.com/user-attachments/assets/7d8f4e90-0575-4098-bb5d-da025a50b20b)

### Acceptance Criteria

- [ ] Any new code is covered by a unit tested.
- [ ] Check code coverage is at least 90%.

### Code of Conduct

- [X] I have checked the [existing issues](https://github.com/Avaiga/taipy/issues?q=is%3Aissue+).
- [ ] I am willing to work on this issue (optional)

**Comment by sahilborse** [comment]
/attempt @FlorianJacta please assign me this issue i can work on it

**Comment by FlorianJacta** [comment]
Here you go @sahilborse

**Comment by sahilborse** [comment]
@FlorianJacta  can you elaborate the issue
As should height be relative to the screen size

**Comment by FlorianJacta** [comment]
See the other issue here too because it is related: https://github.com/Avaiga/taipy/issues/2141

The goal is to change the default height of the metric viz element

**Comment by nehaaprasad** [comment]
Hello @FlorianJacta 
I want to assign for this issue!
THankyou!

**Comment by Manideep-Maddileti** [comment]
hello @FlorianJacta I want to work on this issue can you please assign this issue to me

**Comment by PentesterPriyanshu** [comment]
@FlorianJacta i am working on this issue assign to me

**Comment by FlorianJacta** [comment]
@naaa760 you are assigned tooo

**Comment by Pranshu-jais** [comment]
Hi @FlorianJacta I want to work on this issue can you please assign this issue to me

**Comment by FlorianJacta** [comment]
Sure, I am assigning you this issue @Pranshu-jais

**Comment by namnguyen20999** [comment]
Hi all,

Thank you for your work on this issue! We wanted to let you know that we are closing this issue as it has been addressed in https://github.com/Avaiga/taipy/issues/2141.

We appreciate your efforts and look forward to your contributions in the future. Let us know if you have any questions!

**Comment by FlorianJacta** [comment]
This doesn't seem to be fixed in Taipy 4.0.2.dev1. However, this is fixed in develop

**Comment by AlexandreSajus** [comment]
Fixed in 4.0.2.dev2!
